src/lib/features/metrics/instance/metrics.ts

```diff
@@ -158,6 +158,7 @@ export default class ClientMetricsController extends Controller {
                                 environment: app.environment,
                                 sdkType: app.sdkType,
                                 sdkVersion: app.sdkVersion,
+                                projects: app.projects,
```

Comment:
this is respecting the newly added projects field in edge for each registered application